starters/gatsby-starter-wordpress-blog/package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Just noticed there were a few things off in the package.json file when checking to see what starters we needed to add to the peril settings — so I updated!